api/app/outcome/models.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+from app import db
+from enum import Enum
+
+class Status(Enum):
+    ACCEPTED = "accepted"
+    REJECTED = "rejected"
+    WAITLIST = "waitlist"
+
+class Outcome(db.Model):
+    id = db.Column(db.Integer(), primary_key = True, nullable = False)
+    event_id = db.Column(db.Integer(), db.ForeignKey('event.id'), nullable = False)
+    user_id = db.Column(db.Integer(), db.ForeignKey('app_user.id'), nullable = False)
+    status = db.Column(db.Enum(Status), nullable = False)
+    timestamp = db.Column(db.DateTime(), nullable = False)
+    latest = db.Column(db.Boolean(), nullable = False)
+    updated_by_user_id = db.Column(db.Integer(), db.ForeignKey('app_user.id'), nullable=False)
+
+    event = db.relationship('Event', foreign_keys=[event_id])
+    user = db.relationship('AppUser', foreign_keys=[user_id])
+    updated_by_user = db.relationship('AppUser', foreign_keys=[updated_by_user_id])
+
+    def __init__(self,
+                 event_id,
+                 user_id,
+                 status,
+                 latest,
```

Comment:
Lets remove latest from the constructor parameters and instead default it to True (`self.latest = True` inside the method body), since whenever someone creates a new Outcome, it is by definition the latest one :)